adage/analyze/api.py

```diff
@@ -342,7 +350,7 @@ def apply_filters(self, request, applicable_filters):
 
 class ParticipationResource(ModelResource):
     node = fields.IntegerField(attribute='node_id', null=False)
```

Comment:
Should node also be a foreign key? Not sure what the implications would be but the parallel construction might be nice.